.github/workflows/gh-pages.yml

```
@@ -0,0 +1,41 @@
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: '0.149.1'
+          extended: true
+
+      - name: Build
+        run: |
+          hugo --minify --baseURL "https://prof-ramos.github.io/sherlockramos-blog/"
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)